samples/CounterApp/CounterApp/CounterApp.fs

```diff
@@ -7,116 +7,6 @@ open Fabulous.DynamicViews
 open Xamarin.Forms
 
 module App = 
-#if CALCULATOR
```

Comment:
A little surprise here to find 2 samples in one.
So I took the liberty to split them in 2 distincts samples :)